Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.8.5
+
+ARG MYBRANCH
```

Comment:
This maybe should have a default value. If someone wants to build the docker image without invoking the CI.

https://docs.docker.com/engine/reference/builder/#default-values

Reply:
done. set to master